### PR TITLE
add attacut to pythainlp/tokenize

### PR DESCRIPTION
**Pull request by bkktimber**
This Pull request refere to issue [#258](https://github.com/PyThaiNLP/pythainlp/issues/258)
change lists:
- add `attacut` tokenizer
- add `.vscode` and `.DS_Store` to `.gitignore`

### CODE REVIEW

**Comment by wannaphongcom** [comment]
It has conflicts that must be resolved.

**Comment by heytitle** [comment]
Everything looks good now except the conflict.
@bkktimber  could you please resolve it?

**Comment by bkktimber** [comment]
I fixed all conflicts with Github GUI. Not sure if I did them correctly? 

please advice. thank you

**Comment by heytitle** [comment]
Your merging looks ok to me. I think you might have to update code syntax as well. Please see details from @pep8speaks.  

In some cases, we can ignore warning. @wannaphongcom  might know when such cases are.

**Comment by wannaphongcom** [comment]
The Travis CI build failed.  https://travis-ci.org/PyThaiNLP/pythainlp/builds/579358344

**Comment by bkktimber** [comment]
I've figured out what happened. I mixed the old codebase (like over a year ago) up with the current one. Just cleaned them up with several commits.

sorry for my carelessly mistake.

File: pythainlp/tokenize/attacut.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+import sys
+try:
```
Comment (wannaphongcom) [inline pythainlp/tokenize/attacut.py]: Remove try except like [deepcut](https://github.com/PyThaiNLP/pythainlp/blob/9822e10b81166349533fa43deced446e759d8848/pythainlp/tokenize/deepcut.py).

**Comment by wannaphongcom** [comment]
Can you add attacut test?

File: pythainlp/tokenize/attacut.py (hunk 1 above)
Comment (bkktimber) [reply pythainlp/tokenize/attacut.py]: done.

**Comment by bkktimber** [comment]
@wannaphongcom how can I add attacut test?

**Comment by wannaphongcom** [comment]
Add attacut in tests/test_tokenize.py

**Comment by bkktimber** [comment]
done

**Comment by wannaphongcom** [comment]
Thank you.

**Pull request merged**